docs/includes/schema-builder/flights_migration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+use MongoDB\Laravel\Schema\Blueprint;
+
+return new class extends Migration
+{
+    protected $connect = 'mongodb';
+
+    public function up(): void
+    {
+      // begin create index
+        Schema::create('flights', function (Blueprint $collection) {
+            $collection->index('mission_type');
+            $collection->index(['launch_location' => 1, 'launch_date' => -1]);
+            $collection->unique('mission_id', null, null, ['name' => 'unique_mission_id_idx']);
```

Comment:
Named arguments are useful here:
```suggestion
            $collection->unique('mission_id', options: ['name' => 'unique_mission_id_idx']);
```